data/incipias/incipias first contact.txt

```diff
@@ -245,6 +239,18 @@ mission "Incipias: Help The Stranded 1"
 			`	As you stow it away, the Quarg speaks once more. "To satisfactorily utilize this device, you must direct it towards the source of the communication, so that it is made able to discern the relevant emissions. Given the complexity of their language, the translator might not be able to adequately translate some aspects of their obscure speech."`
 			`	It then looks you directly in the eyes. "Remember what has been told, human. The stars have no place for those who would act foolishly and thoughtlessly towards their fellow beings." The Quarg then walks away, leaving you alone.`
 				accept
+			label cold
+			`	"It is perhaps a laborious task for an adolescent species such as yourself to comprehend our ways, but we must place the continuation of species above the existence of individuals. If we reveal ourselves to them at this inopportune moment, the shock of such an advanced species appearing in their space will fan the flames of chaos."`
+			choice
+				`	"Then let me help them."`
+					goto initiative
+				`	"You can deal with the consequences later. Help them now!"`
+			`	The Quarg takes on a sour look. "Insisting you are. Understand that our own intervention is out of the question." It sighs. "However, this one can foresee the likelihood of you making a rash and impetuous decision to contact these Incipias further."`
+			choice
+				`	"Sure. Let me do the work."`
+				`	"If you insist on not helping..."`
+			`	The Quarg rapidly clicks its teeth before continuing.`
```

Comment:
```suggestion
			`	The Quarg rapidly clicks his teeth before continuing.`
```

Reply:
We always refer to the Quarg as "its" anywhere in the game.

Reply:
I didn't know. Because they're non-binary?